gatling-ss7/src/main/scala/io/snice/gatling/ss7/engine/Ss7Client.scala

```diff
@@ -146,130 +121,68 @@ class Ss7Client(mapStack: MAPStack,
       reSynchronisationInfo, extensionContainer, RequestingNodeType.mmeSgsn, plmnId, numberOfRequestedVectors, additionalVectorsAreForEPS)
   }
 
-  def handleResponse(requestId: RequestId, status: Status): Unit = {
-    val timeEnd = clock.nowMillis
-    val callback = responseCallbacks.getOrElse(requestId, (status:Status, timeNow: Long) => logger.warn(s"No request stored for response with transaction ID ${requestId.transactionId} and invoke ID ${requestId.invokeId}"))
-    callback.apply(status, timeEnd)
-    responseCallbacks.remove(requestId)
-  }
-
-  def successResponseHandler[T <: MobilityMessage](response: T): Unit = {
-    val requestId = new RequestId(response.getMAPDialog.getLocalDialogId, response.getInvokeId)
-    handleResponse(requestId, OK)
-  }
-
-  override def onErrorComponent(mapDialog: MAPDialog, aLong: lang.Long, mapErrorMessage: MAPErrorMessage): Unit = {
-    logger.debug(s"Map Error Code: ${mapErrorMessage.getErrorCode}")
-    val requestId = new RequestId(mapDialog.getLocalDialogId, aLong)
-    handleResponse(requestId, KO)
-  }
-
-  override def onCancelLocationResponse(cla: CancelLocationResponse): Unit = {
-    logger.debug("onCancelLocationResponse")
-    successResponseHandler(cla);
-  }
-
-  override def onPurgeMSResponse(purgeMSResponse: PurgeMSResponse): Unit = {
-    logger.debug("onPurgeMSResponse")
-    successResponseHandler(purgeMSResponse)
-  }
-
-  override def onUpdateLocationResponse(updateLocationResponse: UpdateLocationResponse): Unit = {
-    logger.debug("onUpdateLocationResponse")
-    successResponseHandler(updateLocationResponse)
-  }
-
-  override def onSendIdentificationResponse(sendIdentificationResponse: SendIdentificationResponse): Unit = {
-    successResponseHandler(sendIdentificationResponse)
-  }
-
-  override def onAuthenticationFailureReportResponse(authenticationFailureReportResponse: AuthenticationFailureReportResponse): Unit = {
-    successResponseHandler(authenticationFailureReportResponse)
-  }
-
-  override def onSendAuthenticationInfoResponse(sendAuthenticationInfoResponse: SendAuthenticationInfoResponse): Unit = {
-    successResponseHandler(sendAuthenticationInfoResponse);
+  def addUpdateLocationRequest(imsi: IMSI, clientDialogMobility: MAPDialogMobility): Long = {
+    val mscNumber = new ISDNAddressStringImpl(AddressNature.international_number, NumberingPlan.ISDN, config.LOCAL_GT)
```

Comment:
If we would like to have a different GT for VLR and SGSN is that enough to modify the `config.LOCAL_GT` parameter?